monero-outreach-docs/en/transcriptions/monero-konferenco-2019/Challenges_With_Open_Software_Revieww.md

```diff
@@ -0,0 +1,68 @@
+
+Answer: So part of the problem with that is that we could call out specific areas of the code that need attention. But they're going to change relatively quickly. So I think it needs to be a living effort, and we need a more flexible approach. I do think that issue tracking is applicable. I think it's a good idea, and it'll be useful. But the people who are involved in that are going to have to be updating that stuff. And in some cases, it's just the case that it's more efficient for whoever notices the issue to just fix it right there, because, honestly, that's usually all it takes. And it's usually just the people who are like, "No, I'm not going to do that," like, "I have more important stuff to do." But actually, they don't, necessarily. I think that we might need to change our priorities a little bit in terms of how we write code. It's like we should always be factoring. ABF, always be factoring.
+
+We actually have more time for questions, if anyone can think of anything else. Okay, we have two over there at least.
+
+Question: So I wonder, as a follow on to that last question, I wonder whether it might be nice to have a dedicated IRC channel, actually, for this because I think one of the problems with new changes that come along, they get mixed up in some conversation, the current kind of things that they worked on as opposed to that taking a step to the side. And of course, many will still contribute into that channel, as well. But it's kind of like a different mindset, because what you're talking about here is really about a mindset change, I think, as much as it is, so, "Yeah, let's have a new IRC channel" and work on that.
+
+Answer: Yeah, so I actually made one, #monero-newlibs, but it's actually because of the fact that this needs to be integrated into our culture in a way. It started to occur to me that maybe it shouldn't be separate, at least not right now. There wasn't enough community awareness at the time, I think, to really make that an effective strategy. But I think that as this-- I mean, more and more people are starting to become involved in this. So I think it could actually make sense. I think that's a good idea. So that's #monero-newlibs. 
+
+There's one more back there.
+
+Question: And so I mean, it's a limited, very limited pool of people. There's a lot of contributors, but I don't think that there's very many people who have a kind of good grasp of the whole codebase, so they have a good perspective on where things where should belong, necessarily. And so, I mean, you mentioned a few people, and I was just curious if you thought--and in your response to questions, you've indicated you think it should it be a kind of group effort...and change of mindset, but I thought--I was just curious if you thought it's worth trying to convince a single person to kind of…
```

Comment:
**You kind of ended on a dark note, yeah the problem is getting worse.** And so I mean, it's a limited, very limited pool of people. There's a lot of contributors, but I don't think that there's very many people who have a kind of **good --grasp for factoring or grasp for the functionality of the codebase--**  of the whole codebase, so they **kind of** have a good perspective on where things where should belong, necessarily. And so, I mean, you mentioned a few people, and I was just curious if you thought **- and in your response to questions, you've indicated you think it should be a kind of group effort, and change of mindset, but I thought - I was just curious if you thought it's worth trying to convince a single person to kind of--**